docs/FAQs.md

```diff
@@ -3,10 +3,63 @@ sidebar_position: 7
 ---
 # FAQs
 
-### Contact and Getting Support
+### How can I submit my data?
+You need to first register to [nmrXiv](https://nmrxiv.org/) with your email id and then submit your data via the automated submission pipeline. Read the [submission guides](/docs/category/submission-guides) to get more understanding.
 
-Did you find any bug or want to raise a request for new feature? Then please follow the instruction provided in the [Contribution](https://docs.nmrxiv.org/docs/contributing/contributing-to-nmrxiv) section. 
+### Do I need to register before submitting the data to nmrXiv?
+Yes, you need to register via your email id or you can also log in via Single sign-on with your GitHub or Twitter ID.
 
-If you think your issue is urgent and there is any security vulnerability in nmrXiv, then please reach out to our [Helpdesk](https://www.nfdi4chem.de/index.php/helpdesk/). We will get back to you as quickly as possible.
+### What should be the structure of the folder to be uploaded in nmrXiv?
+
+
+### What are the files format currently supported in nmrXiv?
+
+
+### What happens to my data once submitted?
+
+
+### What is a public and private project in nmrXiv?
+**Public** projects(including the underline studies and datasets) are visible and accessible to everyone(even to the non-registered user of [nmrXiv](https://nmrxiv.org/)). You can see all the open projects [here](https://nmrxiv.org/projects) or can also find them in the Projects tab in the left-hand panel of your dashboard.
+
+**Private** projects(including the underline studies and datasets) are only visible and accessible to you or the members of the team and also to the user with whom you explicitly share the project.
+
+### What happens when I make my project public?
+If you make your project public you are making it open for access(including the underline studies and datasets) to everyone(even to the non-registered user of [nmrXiv](https://nmrxiv.org/)), but you can define rights by choosing a license for your project. Once your project is made public you cannot edit, delete or change it to private again.
```

Comment:
when you make your project public, it becomes open for reading access (including the underling studies and datasets) to everyone (including non-registered users of [nmrXiv](https://nmrxiv.org/)), but you can clarify rights of usage by choosing a license for your project. Once your project is made public, you cannot edit, delete or change it to private again.